serverless/serverless
# feat: add deploymentBucket.maxConcurrentUploads configuration

## What did you implement

Closes #7266 

As per https://github.com/serverless/serverless/pull/4677#discussion_r175391794 I ended up making this a provider configuration at the yml rather than an environment variable.

## How can we verify it

```yml
service: serverless-blank

provider:
  name: aws
  runtime: nodejs12.x
  region: us-east-1
  deploymentBucket:
    maxConcurrentUploads: 1

package:
  individually: true

functions:
  hello:
    handler: src/handler.hello
  world:
    handler: src/handler.world
```
(and add the corresponding source files with any content)

After triggering `sls deploy`, observe the logs and you should see firstly:
> Serverless: Uploading service hello.zip file to S3 (24.47 MB)...
then later (not at the same time as the previous one):
> Serverless: Uploading service world.zip file to S3 (24.47 MB)...

## Todos


**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
